syntax = "proto3";
package mars.stn;


option java_package = "win.liyufan.im.proto";
option java_outer_classname = "ConversationOuterClass";

//option objc_class_prefix = "LYF";


enum ConversationType {
    ConversationType_Private = 0;
    ConversationType_Group = 1;
    ConversationType_System = 2;
    ConversationType_ChatRoom = 3;
    ConversationType_Command = 4;
}

message Conversation {
  ConversationType type = 1;
  string target = 2;
}
